app/3.0/start/install-3-0.md

```diff
@@ -1,87 +1,107 @@
 
-To create an application template and install Polymer automatically, you can [use the Polymer CLI](#use-cli).
+Polymer elements need to be unique, so we recommend using Yarn's `--flat` install option if possible to install your project's dependencies. This prevents your project from attempting to install multiple versions of the same element.
 
-To start a project from scratch, you can [install Polymer with Bower](#use-bower).
+To set up a development environment for Polymer 3.0: 
 
-### Use the Polymer CLI to create an application template and install Polymer {#use-cli}
+* [Install Git](#git).
+* [Install npm and Node.js](#node).
+* [Install Yarn](#yarn).
+* [Install the Polymer CLI](#cli).
+* Optionally, [Download and serve a sample Polymer 3.0 app](#app).
 
-The Polymer CLI requires Node.js, npm, Git and Bower. For full installation instructions, see [the Polymer CLI documentation](../docs/tools/polymer-cli).
+### Install Git {#git}
 
-1. Install the Polymer CLI.
+[Download and run a Git installer for your operating system](https://git-scm.com/download/). 
 
-    ```bash
-    npm install -g polymer-cli
-    ```
+[See the Git documentation](https://git-scm.com/book/en/v2/Getting-Started-Installing-Git) for detailed instructions on installing Git.
 
-3. Create a test folder for Polymer 2.0, and switch to it.
+To confirm that Git is correctly installed:
 
-    ```bash
-    mkdir polymer-20-test
-    cd polymer-20-test
-    ```
+```bash
+git --version
+```
 
-4. Initialize your project.
+### Install npm and Node.js {#node}
 
-    ```bash
-    polymer init
-    ```
+[Download and run a Node.js installer for your operating system](https://nodejs.org/en/download/). This will install npm as well. 
 
-5. Select `polymer-2-application`.
+For more detailed instructions on installing npm and Node.js, see the [npm and Node.js install documentation](https://docs.npmjs.com/getting-started/installing-node).
 
-6. Serve your project.
+**Install a current or active LTS version of Node.js**. Polymer requires a current or active LTS version of Node.js. See our [Node.js support page](/{{{polymer_version_dir}}}/docs/tools/node-support) for more information.
+{ .alert}
 
-    ```bash
-    polymer serve
-    ```
+To confirm that npm is correctly installed:
 
-### Install Polymer from Bower {#use-bower}
+```bash
+npm --version
+```
 
-1. Install Bower.
+To confirm that Node.js is correctly installed:
 
-    ```bash
-    npm install -g bower
-    ```
+```bash
+node --version
+```
 
-2. Install the Polymer CLI
+### Install Yarn {#yarn}
 
-    The Polymer CLI requires Node.js and npm as well as Bower. For full installation instructions, see [the Polymer CLI documentation](../docs/tools/polymer-cli).
+For Mac and Linux users, an easy way to install Yarn is to run the Yarn install script:
 
-    ```bash
-    npm install -g polymer-cli
-    ```
+```bash
+curl -o- -L https://yarnpkg.com/install.sh | bash
```

Comment:
I have grave doubts about installing anything this way. Not that npm is a lot safer, but I'd be inclined to tell people to npm install this. 

/cc @justinfagnani @kevinpschaaf

Reply:
meh I just copied what it says in the Yarn docs.. I should probably just link to them

Reply:
doesn't matter since removing yarn